src/main/java/seedu/workbook/logic/parser/AddCommandParser.java

```diff
@@ -46,9 +48,13 @@ public AddCommand parse(String args) throws ParseException {
         Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         Stage stage = ParserUtil.parseStage(argMultimap.getValue(PREFIX_STAGE).get());
+        // todo extract method out to parse optional and parse mandatory
+        DateTime dateTime = argMultimap.getValue(PREFIX_DATETIME).isPresent()
+                ? ParserUtil.parseDate(argMultimap.getValue(PREFIX_DATETIME).get())
+                : new DateTime("");
         Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
```

Comment:
Good changes over here when adding an internship. I also feel that if the DateTime attribute is not present we should just use an empty "" string instead 👌.